### ISSUE TITLE
About phrasal quality estimator

### ISSUE THREAD

**Issue by isolet** [shangjingbo1226/AutoPhrase#71]
The original paper says 

> Q(·), also known as the phrase quality estimator, is initially learned from data based on statistical features^4

and refers to github for more detail in the footnote. 

Are the definition and calculation of phrasal quality estimator the same as the one in ` Liu, J., Shang, J., Wang, C., Ren, X., & Han, J. (2015). Mining quality
  phrases from massive text corpora. In , Proceedings of the 2015 ACM SIGMOD
  International Conference on Management of Data (pp. 1729–1744). ` ?

Thanks.

**Comment by remenberl** [comment]
Yes. We didn't change the quality estimator implementation related features. However, the ground truth labels are from different sources for the two approaches. Moreover, we train multiple classifiers instead of one for autophrase to reduce noise in the training data.
